<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_num_patient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Numéro patient</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_num_patient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelErreur_existe">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ce numéro de patient existe déjà</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelErreur_existe_pas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ce numéro de patient n'existe pas</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelErreur_num_patient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Entrer un nombre composé de 7 chiffres dont le premier est pair</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom_patient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom du patient</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom_patient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_prenom_patient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prénom du patient</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenom_patient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelErreur_nom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Alphabétique de longueur maximale égale à 50 lettres</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelErreur_prenom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Alphabétique de longueur maximale égale à 50 lettres</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_naissance">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date de naissance du patient</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelErreur_date_naissance">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>La date doit être inférieure ou égale à la date du jour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ajouter_patient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ajouter patient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimer_patient">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Supprimer patient</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
